### PR TITLE
Fix #1612

### LINKED ISSUE #1612: Credits: Add New ST member
@GulpTea needs to be added to the credits (I'm not sure if this is too early to add them to credits, I mean they literally joined 4 days ago).
If this gets help-wanted, I can do this.

### PR DESCRIPTION
**Pull request by VutonDesign**
Updates the credits page as requested in #1612.

### CODE REVIEW

File: src/views/credits/credits.jsx [hunk 1]
```diff
@@ -157,28 +157,18 @@ var Credits = React.createClass({
...
-
-                    <li>
                         <img src="//cdn.scratch.mit.edu/get_image/user/373646_170x170.png" alt="Dalton Avatar" />
                         <span className="name">Dalton Miner</span>
                     </li>
-
+                    
+                    <li>
+                        <img src="//cdn2.scratch.mit.edu/get_image/user/default_60x60.png" alt="Joel Avatar" />
+                        <span className="name">Joel Gritter</span>
```
Comment (thisandagain) [inline src/views/credits/credits.jsx]: Members on the credits page should be in alphabetical order by last name.

File: src/views/credits/credits.jsx [hunk 2]
```diff
@@ -224,7 +214,11 @@ var Credits = React.createClass({
                 Tammy Stern,
                 Lis Sylvan,
                 Claudia Urrea,
-                Oren Zuckerman
+                Oren Zuckerman,
+                Connor Hudson,
```
Comment (thisandagain) [inline src/views/credits/credits.jsx]: Same here. Please put in alphabetical order by last name.

File: src/views/credits/credits.jsx (hunk 1 above)
Comment (VutonDesign) [reply src/views/credits/credits.jsx]: Ah, That makes sense 👍

File: src/views/credits/credits.jsx (hunk 2 above)
Comment (VutonDesign) [reply src/views/credits/credits.jsx]: On it!

**Comment by VutonDesign** [comment]
@thisandagain That should be it sorted out now.

File: src/views/credits/credits.jsx [hunk 3]
```diff
@@ -223,8 +216,9 @@ var Credits = React.createClass({
                 Jay Silver,
                 Tammy Stern,
                 Lis Sylvan,
+                Hanako Tjia
                 Claudia Urrea,
-                Oren Zuckerman
+                Oren Zuckerman,
```
Comment (thisandagain) [inline src/views/credits/credits.jsx]: Please do not add a comma to the end.

Comment (VutonDesign) [reply src/views/credits/credits.jsx]: Ooops

Comment (VutonDesign) [reply src/views/credits/credits.jsx]: Fixed 😄

**Comment by VutonDesign** [comment]
@thisandagain I can't seem to find any Trailing spaces, But Travis is still failing.

File: src/views/credits/credits.jsx [hunk 4]
```diff
@@ -223,6 +216,7 @@ var Credits = React.createClass({
                 Jay Silver,
                 Tammy Stern,
                 Lis Sylvan,
+                Hanako Tjia
```
Comment (thisandagain) [inline src/views/credits/credits.jsx]: Missing a comma here

Comment (VutonDesign) [reply src/views/credits/credits.jsx]: Fixed 👍

File: src/views/credits/credits.jsx [hunk 5]
```diff
@@ -155,20 +155,15 @@ var Credits = React.createClass({
...
-
+                    
                     <li>
-                        <img src="//cdn.scratch.mit.edu/get_image/user/14110644_170x170.png" alt="Lily Avatar" />
-                        <span className="name">Lily Kim</span>
+                        <img src="//cdn2.scratch.mit.edu/get_image/user/default_60x60.png" alt="Joel Avatar" />
+                        <span className="name">Joel Gritter</span>
                     </li>
-
+
```
Comment (thisandagain) [inline src/views/credits/credits.jsx]: Your extra whitespace appears to be here.

**Comment by thisandagain** [comment]
I'm not sure what editor you use @VutonDesign but for many (e.g. Sublime, Atom, etc.) you can configure it to automatically remove trailing whitespace.

**Comment by VutonDesign** [comment]
@thisandagain I just used the web editor for this, But I'll fix it in atom. 😄

**Comment by GreenBayRules** [comment]
I made another pull requeset #1617 that does the same thing, hopefully travis build will pass on that one

**Comment by VutonDesign** [comment]
@GreenBayRules It failed, I'll just fix it through Atom.

**Comment by GreenBayRules** [comment]
lol this trailing spaces thing is so annoying xD

**Comment by VutonDesign** [comment]
@GreenBayRules Hopefully this is it...

**Comment by GreenBayRules** [comment]
My PR has passed.

**Comment by VutonDesign** [comment]
Not fully yet, One more edit to be tested.

**Comment by VutonDesign** [comment]
Finally lol

**Comment by thisandagain** [comment]
@chrisgarrity Looks like this is ready for re-review. We should squash this PR when we merge.

**Comment by Kenny2github** [comment]
Argh, you can't react to reviews.

Well done @VutonDesign! I remember I went through a similar trial-and-error slope when making my first change to the credits :)

**Comment by VutonDesign** [comment]
@Kenny2github Thanks lol

**Comment by mewtaylor** [comment]
Testing on staging, Chrome 61, OSX 10.12:
* Connor Hudson, Lily Kim, Tauntaun Kim and Hanako Tija are all in the previous members section
* Carolina Kaufman and Joel Gritter are both now in the moderators section

Note: The avatar above `Joel Gritter` is currently the default avatar. I submitted #1639 to point at GulpTea's avatar instead, and will mark this as `QA-OK` once that has been merged and is confirmed to be working

**Comment by mewtaylor** [comment]
The proper avatar is now appearing above the name `Joel Gritter`, marking this as `QA-OK`

**Pull request merged**